_content/essays/jhanas-and-the-lay-disciple_bodhi.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "The Jhānas and the Lay Disciples: According to the Pāli Suttas"
+authors:
+  - bodhi
+external_url: "http://dhammatalks.net/Books10/Bhikkhu_Bodhi_The_Jhanas_and_the_Lay_Disciples.htm"
+source_url: ""
```

Comment:
When the source url is down, use Archive.org:

https://web.archive.org/web/20040812172805/http://www.buddhanet.net/budsas/ebud/ebdha267.htm

Reply:
I put the we archive link in, but the actual link works for me

Reply:
You misunderstood. I meant to keep the dhammatalks link you had as the external url and add the archived buddhanet url as the source.